### PR TITLE
Add ToricVarieties_project to HomalgProject.jl

### PR DESCRIPTION
**Pull request by HereAround**
This should allow to download the packages in the ToricVarieties_project as part of the Homalg_Project in Julia (even though most of the functionality is not yet available).

### CODE REVIEW

File: deps/homalg-project.jl [hunk 1]
```diff
@@ -172,6 +172,11 @@ global PACKAGES_BASED_ON_CAP = [
     "ZariskiFrames",
 ]
 
+##
+global PACKAGES_BASED_ON_TORIC_VARIETIES_PROJECT = [
```
Comment (mohamed-barakat) [inline deps/homalg-project.jl:176]:
```suggestion
global PACKAGES_BASED_ON_TORIC_VARIETIES = [
```

File: deps/homalg-project.jl [hunk 2]
```diff
@@ -186,7 +191,7 @@ List of packages which will be considered by
 * [`UpdateAllPackagesFromHomalgProject`](@ref)()
 * [`RemoveAllPackagesFromHomalgProject`](@ref)()
 """
-global PACKAGES_TO_DOWNLOAD = vcat(PACKAGES_BASED_ON_HOMALG, PACKAGES_BASED_ON_CAP, PACKAGES_NEEDED)
+global PACKAGES_TO_DOWNLOAD = vcat(PACKAGES_BASED_ON_HOMALG, PACKAGES_BASED_ON_CAP, PACKAGES_BASED_ON_TORIC_VARIETIES_PROJECT, PACKAGES_NEEDED)
```
Comment (mohamed-barakat) [inline deps/homalg-project.jl:194]:
```suggestion
global PACKAGES_TO_DOWNLOAD = vcat(PACKAGES_BASED_ON_HOMALG, PACKAGES_BASED_ON_CAP, PACKAGES_BASED_ON_TORIC_VARIETIES, PACKAGES_NEEDED)
```

**Review by mohamed-barakat — CHANGES_REQUESTED** [review]
We drop the suffix `_PROJECT` for better readability.

**Comment by HereAround** [comment]
Ok. Updated.

**Pull request merged**